templates/music/album_detail.html

```diff
@@ -4,6 +4,37 @@
 {% load markdown_deux_tags %}
 {% load music %}
 
+{% block socialmeta %}
+    <!-- Google+ data -->
+    <meta itemprop="name" content="{{ album.name }} by {{ artist.name|upper }}">
+    <meta itemprop="description" content="PerDiem is a platform where you can invest in music. Earn money by discovering and supporting your favorite artists.">
```

Comment:
Do you think it makes sense to add the {{ album.albumbio.bio}} for this section or should it be a generic description?